_data/users.yml

```diff
@@ -1,3 +1,11 @@
+################################################################################
+# Users Page
```

Comment:
Data for the users display on the home page and the detailed users page:

https://trino.io
https://trino.io/users.html